src/ui/common/src/components/workflows/StatusBar.tsx

```diff
@@ -278,28 +278,45 @@ export const WorkflowStatusBar: React.FC<WorkflowStatusBarProps> = ({
   // List of the workflow status items filtered out by category: errors, warnings, logs and checks passed.
   const [listItems, setListItems] = useState<WorkflowStatusItem[]>([]);
 
+  // Ignore Parameters from list of workflow status items
+  const shouldShowStatusItem = (statusItem: WorkflowStatusItem) => {
```

Comment:
This is a tiny nit, but I wonder it might be cleaner to do one filter for the general "stuff we don't care about" and then another filter per-category? Not a huge deal either way.